python/ray/tests/test_actor.py

```diff
@@ -362,6 +362,29 @@ def __init__(self):
     assert actor_class_info[b"module"] == b"ray.tests.test_actor"
 
 
+def test_actor_inheritance(ray_start_regular):
+    class NonActorBase(object):
+        def __init__(self):
+            pass
+
+    # Test that an actor class can inherit from a non-actor class.
+    @ray.remote
+    class ActorBase(NonActorBase):
+        def __init__(self):
+            pass
+
+    # Test that you can't instantiate an actor class directly.
+    with pytest.raises(Exception):
```

Comment:
I tried this and noticed that the error is 

*Exception: Actors methods cannot be instantiated directly. Instead of running 'ActorBase()', try 'ActorBase.remote()'.*

Can you change **Actors methods** -> **Actors**.